src/components/program-progress/styles/_ProgramProgressHeader.scss

```diff
@@ -103,7 +103,6 @@ $btn-color-primary: rgba(6, 86, 131, 100) !default;
       }
 
       .heading {
-        font-family: Inter,Helvetica Neue,Arial,sans-serif;
```

Comment:
Including many of these styles are likely artifacts of folks copying CSS from Figma (not recommended given the use of the Paragon design system). Typography like font-family should only come from Paragon.